research/src/pages/select/select.proposal.mdx

```diff
@@ -183,6 +195,26 @@ and is therefore considered to be invalid.
 
 ### Interaction &amp; transition of states
 
+[Select Usage State Diagram](https://xstate.js.org/viz/?gist=7965135d58118c828140f6e830ca93a1)
+
+![Usage of select within other UI](../../images/statediagram-usage-select.png)
+
+[Select State Diagram](https://xstate.js.org/viz/?gist=24e03b6b21dcb0d1eb7471d6ff9043f3)
+
+![States of a select](../../images/statediagram-select.png)
+
+A `change`event fires on AltUpOREnter, as well as when tabbing off the select when the listbox is open.
+
+### Walk Through Scenario Example
+_Example Table_
+
+| State | Action | Resulting red line change | State change / Event fired
+| ---------- | --------------- | ------- | --------- |
+| collapsed    | Alt+Down      |  | expanded |
```

Comment:
I'm seeing references to `ALT + (up/down)` here, which seems like a Windows specific implementation issue that doesn't map to the documented interaction model for Combobox: https://w3c.github.io/aria-practices/#combobox

Reply:
Good point, I'll remove it since we're talking about web contexts here.

Reply:
@jxlz W3C/WHATWG specifications do have optional capability through Non-normative text or the use of prose that use terms like, "May" or "Should". Each of those of course convey something different than "MUST" which represents something that IS NOT optional.